src/reusableComponents/ImageSlider/index.jsx

```diff
@@ -46,7 +46,7 @@ const ImageSlider = ({ slides, doubleClickHandler }) => {
             delayTime={1000}
             style={{
               width: "100%",
-              aspectRatio: "3/2",
+              height: "100%",
```

Comment:
Hi @Tisha6661 giving variable height will cause issue to grid view of posts as you can see in photo
![image](https://github.com/narayan954/dummygram/assets/77617189/fb4f91a2-8be7-4f8c-9a9f-9307543b631e)
you may either rollback or add different style/fix height of posts in grid view.
There's another option as well, they can showed as canvas as well of variable heights